src/lib/ivy/compile-source-files.ts

```diff
@@ -0,0 +1,168 @@
+import * as ng from '@angular/compiler-cli';
```

Comment:
Most of the below, is coped over from https://github.com/angular/angular-cli/blob/master/packages/ngtools/webpack/src/ivy/plugin.ts